packages/strapi-plugin-graphql/services/build-aggregation.js

```diff
@@ -20,7 +20,7 @@ const { toSDL } = require('./schema-definitions');
  * @returns {Boolean}
  */
 const isPrimitiveType = _type => {
-  const type = _type.replace('!', '');
+  const type = noRequired(_type);
```

Comment:
Can you rename the vars so it is type and nonRequiredType just to make more sense

Reply:
032d3ba83d321900ef5132234210c157f14f45ac